test/commands/test_alias.rb

```diff
@@ -1,7 +1,34 @@
 require 'minitest/autorun'
+require_relative '../test__helper'
+require_relative '../fake_home'
+require_relative '../../lib/zold/commands/alias'
 
-# @todo #279:30min Write unit tests for the alias command.
-#  It should test all the subcommands (set, remove and show)
-#  and any options.
 class TestAlias < Minitest::Test
+  # alias set <wallet> <alias>
+  def test_set_writes_alias_to_the_alias_file
+    FakeHome.new.run do |home|
+      wallet = home.create_wallet
+      assert_raises NotImplementedError do
+        Zold::Alias.new(wallets: home.wallets, log: test_log).run(['set', wallet.id.to_s, 'my-alias'])
+      end
+    end
+  end
+
+  # alias remove <alias>
+  def test_remove_removes_the_alias_from_the_alias_file
+    FakeHome.new.run do |home|
+      assert_raises NotImplementedError do
+        Zold::Alias.new(wallets: home.wallets, log: test_log).run(['remove', 'my-alias'])
```

Comment:
@botanicus it seems to me that you're just running the commands, but not checking whether they've done anything?

Reply:
@ledestin that’s true. I’m trying to figure out how to go about PDD, as per my [reply](https://github.com/zold-io/zold/pull/330#discussion_r199915220) to @yegor256 above.